src/vuestic-theme/vuestic-components/vuestic-modal/VuesticModalNew.vue

```diff
@@ -0,0 +1,263 @@
+<template>
+<div
+  v-show="valueProxy"
+  class="vuestic-modal-new__overlay"
+  @click="checkOutside"
+>
+  <transition name="modal">
+    <div
+      class="vuestic-modal-new"
+      :class="computedClass"
+      v-show="valueProxy"
+      :style="{maxWidth, maxHeight}"
+    >
+      <i
+        v-if="closeButton || fullscreen"
+        @click="cancel"
+        class="ion ion-md-close vuestic-modal-new__close"
+      />
+      <div class="vuestic-modal-new__inner" :style="{maxHeight, maxWidth}">
+        <div v-if="title" class="vuestic-modal-new__title">{{title}}</div>
+        <div v-if="hasHeaderSlot" class="vuestic-modal-new__header"><slot name="header"/></div>
+        <div v-if="message" class="vuestic-modal-new__message">{{message}}</div>
+        <div class="vuestic-modal-new__content">
+          <slot/>
+        </div>
+        <div v-if="cancelText !== '' || okText !== ''" class="vuestic-modal-new__actions">
```

Comment:
Well, empty string is sort of falsy anyway. Can we simplify `cancelText || okText`?